assets/wire/components/WireApp.tsx

```diff
@@ -211,34 +217,34 @@ class WireApp extends BaseApp {
                                 <SearchSidebar tabs={this.tabs} props={{...this.props}} />
                             }
                         </div>
-                        <div className={mainClassName}
+                        <div
+                            className={mainClassName}
                             ref={this.setListRef}
                         >
-                            <SearchResultsBar
-                                minimizeSearchResults={this.state.minimizeSearchResults}
-                                initiallyOpen={
-                                    Object.values(this.props.searchParams ?? {}).find((val) => val != null) != null ||
-                                    this.props.activeTopic != null ||
-                                    this.props.activeProduct != null ||
-                                    Object.keys(this.props.activeFilter ?? {}).length > 0 ||
-                                    this.props.activeQuery != null
-                                }
-                                showTotalItems={showTotalItems}
-                                showTotalLabel={showTotalLabel}
-                                showSaveTopic={showSaveTopic}
-
-                                totalItems={this.props.totalItems}
-                                totalItemsLabel={totalItemsLabel}
-
-                                saveMyTopic={saveMyTopic}
-                                activeTopic={this.props.activeTopic}
-                                topicType={this.props.context === 'wire' ? this.props.context : null}
-
-                                newItems={this.props.newItems}
-                                refresh={this.props.fetchItems}
-                                setSortQuery={this.props.setSortQuery}
-                                setQuery={this.props.setQuery}
-                            >
+                            {
+                                showFilters && (
+                                    <SearchResultsBar
+                                        minimizeSearchResults={this.state.minimizeSearchResults}
+                                        initiallyOpen={showFilters}
```

Comment:
I think it's better to leave it since it allows for more control. In the future we might want to change the requirements again so it's intially closed in some cases.